packages/angular_devkit/build_optimizer/src/helpers/ast-utils.ts

```diff
@@ -41,3 +41,15 @@ export function hasPureComment(node: ts.Node): boolean {
 
   return !!leadingComment && leadingComment.some(comment => comment.text === pureFunctionComment);
 }
+
+export function isHelperName(name: string): boolean {
+  // TODO: there are more helpers than these, should we replace them all?
```

Comment:
As a follow up, I think it would be useful to add a one time initialization that would import “tslib” and add all the exported helpers to a set and use that for checks here.  That way it wouldn’t constantly have to be synced.  Especially considering it’s new use in this PR.

Reply:
Thanks for the super fast review. I agree.